supporting_content/columns/region.md

```diff
@@ -1,4 +1,4 @@
-# Column: Region
+# Column: Region ID
```

Comment:
This can't be accepted directly because I think it's adding the table a second time (and we need line 6 to be replaced with line 7. I will fix this directly.